docs.md

```diff
@@ -9,14 +9,18 @@ This document describes the public API available to library consumers.
   - [1.1 - TypeScript](#11---typescript)
 - [2.0 - Interfaces](#20---interfaces)
   - [2.1 - IInsets](#21---iinsets)
-  - [2.2 - IInsetCallbackFunc]()
-- [3.0 - Insets](#30---insets)
-  - [3.1 - addListener](#31---addlistener)
-  - [3.2 - removeListener](#32---removelistener)
-  - [3.3 - getInsets](#33---getinsets)
+  - [2.2 - IInsetCallbackFunc](#22---iinsertcallbackfunc)
+  - [2.3 - IInsetConfiguration](#23---iinsetconfiguration)
+  - [2.4 - InsetMask](#24---insetmask)
+- [3.0 - Inset](#30---inset)
+  - [3.1 - create](#31---create)
+  - [3.2 - free](#32---free)
+  - [3.3 - addListener](#33---addlistener)
+  - [3.4 - removeListener](#34---removelistener)
+  - [3.5 - getInsets](#35---getinsets)
```

Comment:
Missing docs on `getInset`

Reply:
Also `getId`

Reply:
Doesn't talk about static free vs not-static free APIs.

Reply:
I intentionally omitted `getID` since it's not really valuable to end users.

I'll turn rename getInsets, we don't need to document the depecated method.


The create/free APIs are noted as static APIs.

Reply:
Also corrected `IInsets` symbol in the docs (depluralized)

Reply:
```
    public async free(): Promise<void> {
        return await Inset.free(this);
    }
```
There's 2 free APIs, a non-static and a static version. I don't see where the docs note this detail.

Reply:
it's not important enough to mention. The key takeaway is when it's done being used it should be freed and that is documented.

Documenting every single way to free something adds bloat to the documentation which should be clear and concise. There's not much value noting the convenience method.

A better place for that is a reference document, which we don't have.